packages/starlight/package.json

```diff
@@ -180,15 +180,15 @@
     "./style/markdown.css": "./style/markdown.css"
   },
   "peerDependencies": {
-    "astro": "^4.2.7"
+    "astro": "^4.4.5"
   },
   "devDependencies": {
     "@astrojs/markdown-remark": "^4.2.1",
     "@playwright/test": "^1.43.1",
     "@types/node": "^18.16.19",
-    "@vitest/coverage-v8": "^1.3.1",
-    "astro": "^4.3.5",
-    "vitest": "^1.3.1"
+    "@vitest/coverage-v8": "^1.4.0",
+    "astro": "^4.4.5",
+    "vitest": "^1.4.0"
```

Comment:
The Vitest update is required due to https://github.com/withastro/astro/pull/10120 shipped in Astro `v4.4.0`.